src/test/tests/translators.test.ts

```diff
@@ -1,12 +1,61 @@
 import { assert, expect } from 'chai';
-import { ICve, IIssue, IReference, IVulnerableComponent } from 'jfrog-client-js';
+import { ICve, IGraphCve, IIssue, IReference, IVulnerableComponent } from 'jfrog-client-js';
+import { ICve as WebICve, IApplicableDetails } from 'jfrog-ide-webview';
+import { LogLevel } from '../../main/log/logManager';
 import { IIssueCacheObject } from '../../main/types/issueCacheObject';
 import { Translators } from '../../main/utils/translators';
 
 /**
  * Test functionality of @class Translators.
  */
 describe('Translators Tests', () => {
+    it('toAnalyzerLogLevel', async () => {
+        [
+            { extLevel: 'DEBUG', analyzerLevel: 'debug' },
```

Comment:
Let's use the terms "input", "expected", "actual", "output", etc... to make the tests more clear. We would like to show what are the inputs and what are the expected outputs of the tests.
For example 
extLevel -> inputLevel
analyzerLevel -> expectedLevel